### PR TITLE
Add new supported countries to VPN landing page (Fixes #11213)

### LINKED ISSUE #11213: Add new supported countries to VPN landing page
## Description

Launching **March 8**.

[Launch plan](https://docs.google.com/document/d/1emFAlQB1U9KY--tYGHK3NUvZfTnQZnbcFnTZOwt7Kvg/edit?usp=sharing) (Mozilla access only)

- [ ] Add 2 new supported countries to the VPN landing page (see launch plan).
- [ ] Both countries will use existing stripe IDs for English/Euros. 
- [ ] Update landing page strings listing supported countries to include new countries.
- [ ] Add everything behind a feature switch.
- [ ] Needs to be on www-dev.allizom.org for QA testing by March 1st.
- [ ] New strings need to be in Pontoon by March 1st.

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
Adds 2 new supported countries to the VPN landing page behind `SWITCH_VPN_WAVE_V`

## Issue / Bugzilla link
#11213

## Testing

`SWITCH_VPN_WAVE_V=on`

- [x] http://localhost:8000/en-US/products/vpn/?geo=se should display pricing in Euros (clicking through to payment will be in English).
- [x] http://localhost:8000/en-US/products/vpn/?geo=fi should display pricing in Euros (clicking through to payment will be in English).
- [x] http://localhost:8000/en-US/products/vpn/?geo=cn should list the two new countries in availability copy.
- [x] http://localhost:8000/en-US/products/vpn/?geo=cn should mention "Available in 17 countries now. More regions coming soon".

`SWITCH_VPN_WAVE_V=off`

- [x] http://localhost:8000/en-US/products/vpn/?geo=se should display waitlist
- [x] http://localhost:8000/en-US/products/vpn/?geo=fi should display waitlist
- [x] http://localhost:8000/en-US/products/vpn/?geo=cn should not list the two new countries in availability copy.
- [x] http://localhost:8000/en-US/products/vpn/?geo=cn should mention "Available in 15 countries now. More regions coming soon".

### CODE REVIEW

File: bedrock/products/templates/products/vpn/landing.html [hunk 1]
```diff
@@ -81,7 +81,7 @@
               </a>
 
               <p class="availability-copy">
-                {{ ftl('vpn-shared-available-countries-v4', fallback='vpn-shared-available-countries-v3') }}
+                {{ ftl('vpn-shared-available-countries-v5', fallback='vpn-shared-available-countries-v4') }}
```
Comment (maureenlholland) [inline bedrock/products/templates/products/vpn/landing.html:84]: **issue (blocking):** I can see waitlist copy and Finland in available countries list on http://localhost:8000/en-US/products/vpn/?geo=fi (same with ?geo=se & Sweden)
Screen Shot 2022-02-21 at 11 56 26 AM


**suggestion:** move this string behind the switch as well

**Review by maureenlholland — CHANGES_REQUESTED** [review]
One blocking issue on the available countries list in landing page. Otherwise, this is looking like a well-oiled machine - nice work!

File: bedrock/products/templates/products/vpn/landing.html (hunk 1 above)
Comment (alexgibson) [reply bedrock/products/templates/products/vpn/landing.html:84]: Ah, nice catch!

**Comment by alexgibson** [comment]
@maureenlholland thanks for the review, updated!

**Review by maureenlholland — APPROVED** [review]
r+ 💯

**Pull request merged**